src/hub_predtimechart/app/generate_target_json_files.py

```diff
@@ -53,8 +55,41 @@ def main(hub_dir, target_out_dir):
 
 
 def get_target_data_df(hub_dir, target_data_filename):
-    # load the target data csv file from the hub repo for now, file path for target data is hard coded
-    return pl.read_csv(hub_dir / 'target-data' / target_data_filename, null_values=["NA"])
+    """
+    Loads the target data csv file from the hub repo for now, file path for target data is hard coded to 'target-data'.
+    Raises FileNotFoundError if target data file does not exist.
+    """
+    schema = {
+        "location": pl.String,  # to handle the 'US' location (the only one that doesn't parse as Int64)
+        # "date": pl.Date,
+        # "value": pl.Int64
```

Comment:
Any reason not to also include these schema entries?  If they're not to be included, should we just delete these lines?